### PR TITLE
grib-api: init at 1.14.4

### PR DESCRIPTION
**Pull request by knedlsepp**
Add the API for the grib file format.

### CODE REVIEW

File: pkgs/development/libraries/grib-api/default.nix [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+                  openjpeg 
+                  curl     # Used for downloading during make test
+                ];
+  doCheck = true;
+  
+  meta = with stdenv.lib; {
+    homepage = "https://software.ecmwf.int/wiki/display/GRIB/Home";
+    license = licenses.asl20;
+    description = "ECMWF Library for the GRIB file format";
+    longDescription = "The ECMWF GRIB API is an application program interface accessible from C, FORTRAN and Python programs developed for encoding and decoding WMO FM-92 GRIB edition 1 and edition 2 messages.";
```
Comment (jagajaga) [inline pkgs/development/libraries/grib-api/default.nix]: Can you please cut lines by 80 chars? 
Like 
```
fooo
baaa
rrrr
```

**Comment by jagajaga** [comment]
Nice! Please squash commits into one.

**Comment by knedlsepp** [comment]
@jagajaga I'm sorry but I have to admit that I don't know how to do this. Could you point me to the correct command to squash the commits?

**Comment by knedlsepp** [comment]
@jagajaga I think this should now be squashed into one. I (hopefully correctly) used `git rebase -i`.

**Comment by jagajaga** [comment]
Nice! Thank you.

**Pull request merged**